apps/antalmanac/src/components/Header/News.tsx

```diff
@@ -0,0 +1,153 @@
+import { Fragment, useCallback, useState } from 'react';
+import {
+    Badge,
+    Box,
+    Button,
+    Divider,
+    List,
+    ListItem,
+    ListItemIcon,
+    ListItemText,
+    MenuItem,
+    Paper,
+    Popover,
+    Skeleton,
+    Tooltip,
+    Typography,
+} from '@mui/material';
+import { NewspaperOutlined as NewspaperIcon } from '@mui/icons-material';
+import analyticsEnum, { logAnalytics } from '$lib/analytics';
+import { trpc } from '$lib/trpc';
+
+interface NewsPopoverProps {
+    anchorEl: Element | undefined;
+    setAnchorEl: React.Dispatch<React.SetStateAction<Element | undefined>>;
+    setShowDot: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+function NewsPopover(props: NewsPopoverProps) {
+    const { anchorEl, setAnchorEl, setShowDot } = props;
+
+    const query = trpc.news.findAll.useQuery();
```

Comment:
A future TODO is to change this to an infinite query, and have it automatically load more news items as the user scrolls

```suggestion
    const query = trpc.news.findAll.useInfiniteQuery();
```